README.md

```diff
@@ -1,74 +1,37 @@
-# Project Name
+# Building-Blocks---Enumerables
 
-> One paragraph statement about the project.
+Enumerables
 
-![screenshot](./app_screenshot.png)
+Code on Ruby
 
-Additional description about the project and its features.
+![screenshot](./images/Code.PNG)
 
-## Built With
-
-- Major languages,
-- frameworks,
-- technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+## Getting the Repo
 
-### Prerequisites
+git clone https://github.com/shubham14p3/Building-Blocks---Enumerables.git
 
-### Setup
-
-### Install
-
-### Usage
-
```

Comment:
You should include this section and explain to the user how to check your methods on the irb.

Reply:
One I am able to complete the Testing phase program, I will added it, till now since I have not completed it, so I don't want to include it now.